### ISSUE TITLE
403 Authorization error on =>"$request->findModelQuery()->first();"

### ISSUE THREAD

**Issue by khalil-htme** [laravel/nova-issues#2010]
- Laravel Version: 6+
- Nova Version: 2.4

### Description

    public function actions(Request $request){

        return [

        (new Actions\Approve)->canSee(function ($request) {

            $model = $request->findModelQuery()->first();  

            return !empty($model);
        
        })->canRun(function ($request, $user) {
            return true;
            
        });
       ];
     }

Is throwing a 403 authorization error!! this only happened after 2.4 upgrade, and now my entire system is frozen!

basically you are not allowed to pull the first model anymore.

`
  $model = optional($request->findModelQuery()->first()); 
  $model->test;
`

will also throw 403 error... R.I.P

**Comment by PieterxJan** [comment]
@khalil-htme Hey man, I'm experiencing the same issue. Did you find a solution to this issue?

**Comment by thienkimlove** [comment]
For me 
`->canSee(function ($request) {
            return optional($request->findModelQuery()->first())->isOnTrial();
 })
`
which in document is not working in case `exceptOnIndex()`.

What i fix is using 

`
->canSee(function ($request) {
                    return $this->resource instanceof \App\Order && $this->resource->canMakeSuccessLastMonth();
                })
`
